lit/setup-and-operations/install.lit

```diff
@@ -68,6 +68,232 @@ page.
   different machine than the server.
 }
 
+
+\section{
+  \title{Configuring Auth Providers}{auth-config}
+
+  At startup, you will need to configure any local users or third-party auth
+  providers that you wish to use with Concourse.
+
+  In contrast to previous releases of Concourse, \code{users} are now central
```

Comment:
This type of wording may be better off in the release notes or a blog post - first-time users reading the docs don't really need to know how it's different. The rest of the para is good info to know, I'd just state it outright and we'll cover the "how it's different" in places people will look when upgrading.